src/SFML/Graphics/Shader.cpp

```diff
@@ -311,7 +311,7 @@ namespace sf
 struct Shader::Impl
 {
     using TextureTable = std::unordered_map<int, const Texture*>;
-    using UniformTable = std::unordered_map<std::string, int, StringHash, StringEq>;
+    using UniformTable = std::unordered_map<base::StringView, int, StringHash, StringEq>;
```

Comment:
I believe this change is incorrect, as the `UniformTable` does need to own the strings, while `getUniformLocation` can accept a string view as a parameter. The presence of `StringHash` and `StringEq` above should allow the unordered map to be looked up via a string view even if it contains a string. 

In fact, this code compiles everywhere I've tried (Windows x64, Windows ARM64, Linux x64). Could you please share the error message you're getting and more information about the compiler you're using?